src/overlays/actors/ovl_Item_B_Heart/z_item_b_heart.h

```diff
@@ -3,12 +3,20 @@
 
 #include "global.h"
 
-struct ItemBHeart;
+typedef enum {
+    /*   0 */ BHEART_PARAM_NORMAL,     // Spawn Parameter for Normal Scale.
+    /*  35 */ BHEART_PARAM_SMALL = 35, // Spawn Parameter for Small Scale.
+} ItemBHeartParams;
+
+#define BHEART_SCALE_SMALL (0.1f)           // Scale for Small Mode
+#define BHEART_SCALE_NORMAL (1.0f)          // Scale value for Normal Mode
```

Comment:
```suggestion
#define BHEART_SCALE_SMALL (0.1f)           // Scale for Small Mode
#define BHEART_SCALE_NORMAL (1.0f)          // Scale for Normal Mode
```
Or do `Scale value` for both